.github/README.md

```diff
@@ -0,0 +1,108 @@
+# E2E Tests on GitHub CI
+
+E2E tests are executed on GitHub CI using the [E2E Tests Reusable Workflow](#about-the-reusable-workflow) or dedicated workflows.
+
+## Automatic workflows
+
+These workflows are designed to run automatically at crucial stages of the software development process, such as on every commit in a PR, nightly or before release.
+
+### PR E2E Tests
+
+Run on every commit in a PR to ensure changes do not introduce regressions.
+
+Link: https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/integration-tests.yml
+
+### Nightly E2E Tests
+
+Conducted nightly to catch issues that may develop over time or with accumulated changes.
+
+Link: https://github.com/smartcontractkit/chainlink/blob/develop/.github/workflows/run-nightly-e2e-tests.yml
+
+### Release E2E Tests
+
+Currently, there are no automatic workflows triggering E2E tests at release. These tests are triggered on-demand by QA as needed.
```

Comment:
that is not 100% true :-) compatibility pipeline is triggered, when a new tag is created and that tag is usually a release tag

Reply:
@Tofel true. I updated the docs